server/config.go

```diff
@@ -66,6 +66,8 @@ type Options struct {
 	DefaultRepositoryConfig *bulldozer.Config `yaml:"default_repository_config"`
 
 	ConfigurationV0Paths []string `yaml:"configuration_v0_paths"`
+
+	DisableUpdateFeature bool `yaml:"disable_update_feature"`
```

Comment:
Thinking out loud here: are there any other features we'd want this for in the future? Would it make sense to create some kind of feature flag configuration section or are individual options fine for the one or two things this might apply to?

Reply:
I think a feature flag section might be interesting. I think the `options` block kind of fits that purpose already?